test/OpenTelemetry.Tests/Logs/LogRecordTest.cs

```diff
@@ -107,6 +114,118 @@ public void CheckStateForUnstructuredLogWithStringInterpolation()
             Assert.Equal(message.ToString(), state.ToString());
         }
 
+        [Fact]
+        public void CheckStateCanBeSet()
+        {
+            var message = $"This should be replaced {0.99}.";
```

Comment:
the message is confusing...something like logger.LogInformation("this does not matter") would clarify intent.